fuse_publishers/include/fuse_publishers/stamped_variable_synchronizer.h

```diff
@@ -261,34 +261,44 @@ struct is_variable_in_pack<T, Ts...>
 
 }  // namespace detail
 
+// NOTE(CH3): ROS_TIME_ZERO should be uninitialized, it's used for getting latest time!
 template <typename ...Ts>
-const ros::Time StampedVariableSynchronizer<Ts...>::TIME_ZERO = ros::Time(0, 0);
+const rclcpp::Time StampedVariableSynchronizer<Ts...>::ROS_TIME_ZERO =
+  rclcpp::Time(0, 0, RCL_ROS_TIME);
```

Comment:
Seems like `rclcpp::Time::is_valid()` would eliminate the need for these constants since it would give downstream users a different way to check if time is valid or not.

Reply:
From the earlier comment

> Wouldn't it be more explicit that it's a comparison to Time(0, 0) here so readers of the code know that it's something like what TF2 is doing?

I think we should keep it, but if we should remove it I can do that too!

Reply:
https://github.com/locusrobotics/fuse/pull/283/commits/bdedc95997d9e5fe34614bdeace81f21fabb654f